src/fsharp/TastPickle.fs

```diff
@@ -995,7 +996,9 @@ and p_ILCallConv (Callconv(x, y)) st = p_tup2 p_ILHasThis p_ILBasicCallConv (x,
 
 and p_ILCallSig x st = p_tup3 p_ILCallConv p_ILTypes p_ILType (x.CallingConv, x.ArgTypes, x.ReturnType) st
 
-and p_ILTypeRef (x: ILTypeRef) st = p_tup3 p_ILScopeRef p_strings p_string (x.Scope, x.Enclosing, x.Name) st
+and p_ILTypeRef (x: ILTypeRef) st = 
+    // TODO: Add sanity check.
```

Comment:
What is a sanity check here, considering there wasn't a check in the previous implementation?

Reply:
I think the sanity check was already put in place with the p_ILScopeRef